### DISCUSSION TITLE
Idea to solve the pain of moving subs

### DISCUSSION THREAD

**Discussion by adamkad1** [FakeFishGames/Barotrauma#6761] [Ideas and suggestions]
You probably know that moving stuff from old sub to new sub is pain right? So, I suggest the submarine swapping console to have a infinite (or very large) storage near it (or in it), with buttons that can grab items from every storage unit on the sub, and another to put stuff into random storage units that the stuff fits in.

**Comment by GenericGoose** [comment]
I'd rather have a store-like UI where you select a target sub and then have the list of all items and choose the ones you want to move, then they just spawn in the other sub's cargo when it loads
